docs/versioned/getting-started/index.md

```diff
@@ -0,0 +1,95 @@
+# Getting Started with Knative
+
+Welcome! This guide helps you get started with Knative based on your goals and experience level.
```

Comment:
We should mention here that this is targeted at developers and users evaluating using Knative.  For installation, we should link to the "Administration Overview" at `docs/versioned/administer/README.md`